lib/AST/Decl.cpp

```diff
@@ -2597,6 +2597,12 @@ static AccessLevel getTestableOrPrivateImportsAccess(const ValueDecl *decl) {
   return AccessLevel::Public;
 }
 
+/// Return the access level of an internal or public declaration
+/// that's been testably imported.
+static AccessLevel getTestableOrPrivateImportsAccess(const ValueDecl *decl) {
+  return getMaximallyOpenAccessFor(decl);
+}
```

Comment:
I think you don't really need this helper anymore. It'd be clear enough in context to just use the new name at the call sites.

Reply:
Sounds good! I wasn’t sure if it would be better to use old one or remove it.